src/dmd/backend/out.d

```diff
@@ -1417,7 +1417,7 @@ version (SCPP)
 {
                 objmod.codeseg(cpp_mangle(funcsym_p),1);
 }
-else static if (TARGET_WINDOS)
+else if (config.exe & EX_windos)
```

Comment:
`version () {} else if` is not the easiest to understand at a glance.

Reply:
Yeah, but it doesn't mess up all the indentation. See my other remarks on indentation.